bin/python_scripts/tests/test_fix-organograms-s3-filenames.py

```diff
@@ -0,0 +1,56 @@
+#
```

Comment:
In general, python filenames shouldn't have dashes in, otherwise they can't be imported.

Reply:
Though tests won't really be imported but will update